plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelquery/elements/BazelqueryTokenType.kt

```diff
@@ -0,0 +1,10 @@
+package org.jetbrains.bazel.languages.bazelquery.elements
+
+import com.intellij.psi.tree.IElementType
+import org.jetbrains.bazel.languages.bazelquery.BazelqueryLanguage
+
+class BazelqueryTokenType(debugName: String) : IElementType(debugName, BazelqueryLanguage) {
```

Comment:
nit: it bugs me to see query is not camelcased in these class names, `BazelQueryTokenType` and `BazelQueryLanguage` should be better.